plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx

```diff
@@ -31,6 +32,10 @@ export const PaymentGateways = ( {
 	setupPlugin,
 }: PaymentGatewaysProps ) => {
 	const setupLivePayments = () => {};
+	const [ storeCountry, setStoreCountry ] = useState( 'US' );
+	const countryOptions = Object.entries( window.wcSettings.countries || [] )
```

Comment:
What do you think about memoizing this since it's a pretty long list that doesn't change?

Reply:
Great suggestion! I’ve updated the code to include memoization for the list.